### ISSUE TITLE
no encryption

### ISSUE THREAD

**Issue by zzgulu** [dscottbuch/cTiVo#188]
In both 2.5 and 2.4.4 versions:
after downloading and detecting ads it skips from encryption phase and jumps back to downloading again. When I tried "Always use Tivo's TS format" in v2.5 , encryption started and finished but there was no audio at the end - (Tivo Roamio) -

**Comment by mackworth** [comment]
So that sounds exactly like the H264 transition.  Which Format did you use with 2.5?  Can you set Debug to Major run a show, and then post a log?

**Comment by zzgulu** [comment]
Thank you very much for the quick response. The format was iPhone, I tried iPhone super resolution as well which didn't help in 2.5. However, switching back to iPhone super resolution fixed the sound issue in 2.44 (570) - Does the size of the show has anything to do with this? The reason I used 2.5 was due to the fact that I used ctivo for the first time on an one hour "The Kitchen" show with 6Gb in size (HD) where it never went into the encoding phrase after cutting the ads. 
It seems using the iPhone super resolution in 2.4.4 fixed my issues (encoding and sound) for all the shows 
Where can I locate the log file? 

Thanks again
